app/assets/stylesheets/components/auth-form.scss

```diff
@@ -1,26 +1,27 @@
+@import "base/colors";
 
 .auth-form {
+  background: $auth-background;
+  border: 1px solid $field-border;
+  border-radius: 5px;
   display: flex;
   flex-direction: column;
   margin: 30px auto;
-  background: #f8fbfd;
-  border: 1px solid #dae1e3;
-  border-radius: 5px;
   max-width: 450px;
   padding: 45px;
+  position: relative;
   text-align: left;
   width: 100%;
-  position: relative;
 
   .avatar-placeholder {
-    height: 80px;
-    width: 80px;
+    background-color: $avatar-background;
+    border: 3px solid $white;
     border-radius: 50%;
+    left: calc(50% - 40px);
```

Comment:
Properties should be ordered background-color, border, border-radius, -webkit-box-shadow, height, left, position, top, width